### PR TITLE
test(web): Adding tests for life events

### LINKED ISSUE #7574: Test Case: Life Events are visible on front page and can be clicked

### PR DESCRIPTION
**Pull request by pshomov**
# ...

Closes #7574 

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by kristofer-lp8f — COMMENTED** [review]
Looks nice.

We talked a bit about not using `data-cy`, so we don't pin our selectors on the web pages to Cypress, in case we change to a different front-end testing framework. I don't have strong opinions on this, though.

**Comment by kristofer-lp8f** [comment]
You could verify that all life-events are clickable with [`.each`](https://docs.cypress.io/api/commands/each)

File: apps/system-e2e/src/integration/web.spec.ts [hunk 1]
```diff
@@ -11,4 +11,8 @@ describe('web', () => {
     cy.visit('/')
     cy.contains('Að eignast barn')
   })
+  xit('should have life events', () => {
```
[the remark this answers is not in the document]
Comment (pshomov) [reply apps/system-e2e/src/integration/web.spec.ts]: No, `xit` is to disable the test. I think same as `it.not` or something like that.

**Review by darrikonn — APPROVED** [review]
LGTM

**Review by kksteini — APPROVED** [review]

**Pull request merged**